### PR TITLE
Update the base project to make it work

### PR DESCRIPTION
**Pull request by nahuelgl**
## Summary ##

**Trello card**: https://trello.com/c/TCAMnHHV/3-baseproject-script-para-rename-del-proyecto & https://trello.com/c/pnwjNbSq/4-baseproject-arreglar-bug-con-carthagecache-que-esta-borrando-frameworks-necesarios-al-prunear

Fixed the carthage_cache whitelist
Add a rename script to streamline the kickoff

### CODE REVIEW

File: rename [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    echo " ✔ Ack already installed!"
+else
+    echo ""
+    echo " → Installing Ack..."
+    brew install ack
+fi
+
+echo ""
+echo " → Finding and renaming files..."
+#Find and reame every old_name appearance
```
Comment (DiegoQuiros94) [inline rename]: rename*

File: rename [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+find . -name "$old_name*" -print0 -depth | xargs -0 rename -S "$old_name" "$new_name"
+ack --literal --files-with-matches "$old_name" | xargs sed -i "" "s/$old_name/$new_name/g"
+
+#Verify every old_name appearance has been renamed
+old_name_appeareances=$(ack --literal "$old_name")
+if [ "$old_name_appeareances" == "" ]
+then
+    #If this hasn't print out anything, you are good to go!
```
Comment (DiegoQuiros94) [inline rename]: what it it does print something? 🤔

File: rename (hunk 1 above)
Comment (nahuelgl) [reply rename]: details, details

File: rename (hunk 2 above)
Comment (nahuelgl) [reply rename]: https://es.wikipedia.org/wiki/Extinci%C3%B3n_del_contrato_de_trabajo_(Argentina)

Comment (nahuelgl) [reply rename]: I'll add a message hhah

**Pull request merged**